src/plugins/intel_cpu/tests/unit/shape_inference_test/unsqueeze_shape_inference_test.cpp

```diff
@@ -51,6 +54,24 @@ TEST_F(UnsqueezeStaticShapeInferenceAssertTest, empty_axes) {
     }
 }
 
+TEST_F(UnsqueezeCustomStaticShapeInferenceAssertTest, wrong_pattern) {
+    const auto arg = std::make_shared<op::v0::Parameter>(element::f64, PartialShape{-1, -1});
+    const auto axes = std::make_shared<op::v0::Parameter>(element::i64, PartialShape{1});
+
+    const auto op = std::make_shared<op::v0::Unsqueeze>(arg, axes);
+
+    input_shapes = ShapeVector{{5, 6}, axes->get_shape()};
+
+    int64_t axes_data[] = {3};
+    const auto axes_tensor = std::make_shared<ngraph::runtime::HostTensor>(element::i64, ov::Shape{1}, axes_data);
```

Comment:
Try avoid using `ngraph` namespace if not required. Replace by `HostTensor` or `ov::HostTensor` ?

If applied check similar cases in added code.

Reply:
fixed